packages/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "17.0.0",
+  "version": "18.0.0",
```

Comment:
Can you also do a major version update in the react-components package.json file too?

Reply:
good catch - done!